pkg/core/ca/issuer/issuer.go

```diff
@@ -103,6 +103,25 @@ func newWorkloadTemplate(trustDomain string, tags mesh_proto.MultiValueTagSet, p
 	return template, nil
 }
 
+var maxUint128, one *big.Int
+
+func init() {
+	one = big.NewInt(1)
+	m := new(big.Int)
+	m.Lsh(one, 128)
+	maxUint128 = m.Sub(m, one)
+}
+
+func newSerialNumber() (*big.Int, error) {
+	res, err := rand.Int(rand.Reader, maxUint128)
```

Comment:
is it ok to not use secure random = crypto/rand?

Reply:
The import is: `"crypto/rand"` no?